### ISSUE TITLE
fix the issue that's causing pdk mismatches

### ISSUE THREAD

**Issue by radhe-2022** [efabless/caravel_user_project#228]
https://github.com/The-OpenROAD-Project/OpenLane/issues/1579#issue-1496737410

![openlane error](https://user-images.githubusercontent.com/111482662/208046671-bf601446-296a-4e1b-9eb4-0bfa38730416.png)

@mattvenn @proppy @shalan @mkkassem @rb-efabless

**Comment by xobs** [comment]
Note that if you do as it suggests and run `make pdk`, this does not solve the issue.

**Comment by xobs** [comment]
It is, but it's a somewhat scary warning.

My issue with it is that it says to run `make pdk`, but that doesn't actually solve the issue.

**Comment by proppy** [comment]
> It is, but it's a somewhat scary warning.

I agree, I was mostly worrying if the issue was actually blocking the OP or not :)
